modules/update-preparing-list.adoc

```diff
@@ -5,7 +5,7 @@
 [id="update-preparing-list_{context}"]
 = Removed Kubernetes APIs
 
-{product-title} 4.12 uses Kubernetes 1.25, which removed the following deprecated APIs. You must migrate manifests and API clients to use the appropriate API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Kubernetes documentation].
+{product-title} 4.13 uses Kubernetes 1.26, which removed the following deprecated APIs. You must migrate manifests and API clients to use the appropriate API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Kubernetes documentation].
```

Comment:
Can you back out the changes to this file? I have another PR that will take care of everything needed in this file (#55224)